manifests/xsendfile.pp

```diff
@@ -0,0 +1,3 @@
+class pulp::xsendfile {
+  include ::apache::mod::xsendfile
```

Comment:
This should just go in the apache.pp -- https://github.com/Katello/puppet-pulp/blob/master/manifests/apache.pp#L5